specs/electra/beacon-chain.md

```diff
@@ -1139,6 +1151,20 @@ def process_withdrawals(state: BeaconState, payload: ExecutionPayload) -> None:
 
 #### Execution payload
 
+##### New `compute_execution_requests_hash`
+
+*Note*: Computes commitment to the execution layer triggered requests data.
+The computation algorithm is defined by [EIP-7685](https://eips.ethereum.org/EIPS/eip-7685).
+
+```python
+def compute_execution_requests_hash(execution_requests: ExecutionRequests) -> Hash32:
```

Comment:
Alternatively we can do:
```python
def compute_execution_requests_hash(execution_requests: ExecutionRequests) -> Hash32:
    request_list = [execution_requests.deposits, execution_requests.withdrawals, execution_requests.consolidations]
    hash_list = []
    for index, requests in enumerate(requests_list):
        request_type = uint_to_bytes(uint8(index))
        hash_list.append(hash(request_type + serialize(requests)))

    return hash(b''.join(hash_list))
```

This allows to get rid of explicit type bytes in the spec and easy to extend in the future by adding requests to the flat list used for commitment computation